assert/assert.go

```diff
@@ -0,0 +1,43 @@
+// Package assert is a wrapper over testify's assert package.
+package assert
+
+import (
+	"github.com/stretchr/testify/assert"
+)
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 github.com/stretchr/testify/assert.TestingT
+
+// Assertions adds assertions to testify's assert lib.
+type Assertions struct {
+	*assert.Assertions
```

Comment:
I think it is best to name this field explicitly, even though it is legal to not do so, and it has a name implicitly.  It avoids the confusion below in which you explicitly provide a name to this field, and order the fields of this struct differently.

Reply:
I'll add an explicit comment to state that this is done for embedding purposes. If I name it in the type definition, I lose the embedding.